### PR TITLE
Updated setup.md giving it a title

### PR DESCRIPTION
**Pull request by akashagra**

### CODE REVIEW

**Comment by jesmrec** [comment]
Thanks for your contribution @akashagra!. Please sign CLA (the link is above this message) and please rebase the branch against master.

I will suggest you a little improvement. You are adding a title, so i will set such title with bold fond, by adding prefix ### . Notice the difference:

- Setup  

- ### Setup

if @davigonz agrees, we can move forward.

File: SETUP.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Setup Information
```
Comment (davigonz) [inline SETUP.md]: Please use ### to make the text bigger as @jesmrec commented you

**Comment by davigonz** [comment]
Sorry, I approved the changes wrongly the first time. I requested the change that @jesmrec commented above.

**Comment by jesmrec** [comment]
@akashagra did you close accidentally?

**Comment by akashagra** [comment]
Yes i closed it accidentally. I am sorry for that,i have created a new pull request and has done what you suggested

**Comment by jesmrec** [comment]
no worries, we keep the conversation there :)

**Pull request closed without merging**